backend/src/test/resources/queries/edit-queue-item-multiplex-result-mutation

```diff
@@ -1,14 +0,0 @@
-mutation EditQueueItemMultiplexResult($id: ID!, $deviceId: String, $results: [MultiplexResultInput], $dateTested: DateTime) {
```

Comment:
this was moved, not deleted, see [edit-queue-item-multiplex-result-mutation.graphql](https://github.com/CDCgov/prime-simplereport/pull/4243/files#diff-1b9eb6ec38d35dba19f274bb11f7903f2703072d0e1c322aaf3279f613553a88)